assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../keyboard';
+import { useStory } from '../../app';
+
+/**
+ * @param {{current: Node}} ref
+ */
+function useCanvasKeys(ref) {
+  const {
+    actions: {
+      arrangeSelection,
+      clearSelection,
+      deleteSelectedElements,
+      updateSelectedElements,
+    },
+  } = useStory();
+
+  // Return focus back to the canvas when another section loses the focus.
+  useEffect(() => {
+    const container = ref.current;
```

Comment:
Is it better to check if `ref.current` is set before assigning it to a const?

Reply:
No big difference, imho. Any type check is smart enough to realize that the `|null` part of the type is removed after `if (!container) { return }`.